presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -659,6 +659,9 @@ class SystemConfig : public ConfigBase {
   static constexpr std::string_view kPrestoDefaultNamespacePrefix{
       "presto.default-namespace"};
 
+  // Specifies the type of worker pool
+  static constexpr std::string_view kPoolType{"pool-type"};
```

Comment:
Doesn't seem like there is perfect consistency in the naming of config, but we could consider a 2 part name.. announcement.pool-type ? There are other announcement config .